### ISSUE TITLE
Missing Accessor Method in API of ShapeOperation

### ISSUE THREAD

**Issue by jgsuess** [aml-org/amf#1424]
Your issue may already be reported! Please search on Github issues before creating one.

* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] question
  - [ ] other

* **What is the current behavior?**
Constructor method `class amf.shapes.client.platform.model.domain.operations.ShapeOperation.withResponses` should have corresponding accessor method `class amf.shapes.client.platform.model.domain.operations.ShapeOperation.responses()`.

I also cannot find any means in the whole hierarchy to obtain the full list of `responses`. This is despite 
[ShapeOperation](https://github.com/aml-org/amf/blob/develop/documentation/model.md#shapeoperation) showing an array. Is the library unable to provide this information? How can I query the property `http://a.ml/vocabularies/shapes#returns`?

* **If the current behavior is a bug, please provide the minimal steps to reproduce with AMF code. Please don't upload AMF code mixed with yours.**

When using the naming convention to match corresponding accessors for constructors, the resolution fails. It is impossible to retrieve the data. Looking at the source

```scala
 override def response: ResponseType = _internal.responses.head
```
 it is obvious that this is returning an iterator value. However, in `Operation` there is a function to return `responses()`. This means that `ShapeOperation` cannot be inspected, as the parser does not expose.

* **What is the expected behavior?**

provide an explicit method `responses()` that delivers the sequence of responses or remove the `withResponses()` method.

* **What is the motivation/use case for changing the behavior? (for feature requests)**
Automated transcription.

* **Please tell us about your environment:**
  
  - AMF Version: 5
  - AMF Distribution: JVM
  - Operating System: Ubuntu 20.04
  - Link to your project: proprietary

* **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, Gitter, etc)

**Comment by jgsuess** [comment]
I also cannot find any means in the whole hierarchy to obtain the full list of `responses`. This is despite 
[ShapeOperation](https://github.com/aml-org/amf/blob/develop/documentation/model.md#shapeoperation) showing an array. Is the library unable to provide this information? How can I query the property `http://a.ml/vocabularies/shapes#returns`?

**Comment by arielmirra** [comment]
Hi @jgsuess! That's correct, the responses accessor method is missing. In a ShapeOperation, there's always only one response stored, that's why we haven't added the `responses()` method.

Thank you for reporting, I've created an internal issue to track this, with id W-11233146. I'll keep you up to date, it should be available in the AMF release version 5.0.10, next month.

**Comment by arielmirra** [comment]
Hi @jgsuess! This issue has been solved in PR #1454! It'll be available in the next AMF release (coming after 5.0.10). Thank you!
